src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -96,6 +104,26 @@ const TeamMembers = createReactClass({
     });
   },
 
+  addMemberButton(access, orgId) {
+    return access.has('org:write') ? (
+      <Button
+        priority="primary"
+        size="small"
+        className="pull-right"
+        to={`/settings/organization/${orgId}/members/new/`}
+      >
+        <span className="icon-plus" /> {t('Invite Member')}
+      </Button>
+    ) : (
+      <a
```

Comment:
Why no `<Button>` here?

Reply:
great question. this code is just copied from the old stuff, but i'll investigate as I continue working on it.